Doc/c-api/unicode.rst

```diff
@@ -398,7 +398,7 @@ APIs:
    .. versionadded:: 3.3
 
 
-.. c:function:: PyObject* PyUnicode_FromKindAndData(int kind, const void *buffer, \
+.. c:function:: PyObject* PyUnicode_FromKindAndData(int kind, const void *str, \
```

Comment:
This is not a string of `char`s. It is a sequence of 8-, 16- or 32-bit integers, depending of the kind. *buffer* is a good name.